docs/CHANGELOG STYLE GUIDE.md

```diff
@@ -0,0 +1,145 @@
+
+Use emojis and clear keywords to categorize changes:
+- 🆕 New Feature: `🆕`
+- 🎨 Improvement: `🎨`
+- 💄 UI Change: `💄`
+- 🐛 Bug Fix: `🐛`
+- 🔧 Maintenance: `🔧`
+- 🚀 Performance Improvement: `🚀`
+- 📝 Documentation: `📝`
+- ⬆️ Dependency Update: `⬆️`
+- 🗑️ Deprecation/Removal: `🗑️`
+
+---
+## Detailed Entries
+
+### 1. Feature/Change Description:
+Provide a brief and clear description of the change.
+You can use the convention commit from the base but you’ll have to reformat it into a natural language (in the imperative mood) because it fits all contexts, including explaining changes to a stakeholder in person.
+
+You can add reference to a commit, issue or pull request at the end of the same line of a change.
+- Example: `🆕 Chips can now be selectable and closed` (#192)
+### 2.  Notice, Caution and Warning Notes:
+A release might have a separate upgrade guide or blog post that is considered essential reading.
+
+For these purposes a _notice_ must be used. This is a single-sentence paragraph with otherwise arbitrary Markdown content. Adding Markdown emphasis markers is recommended. For example (links omitted):
+```markdown
+## [2.0.0]
+
+_2024-06-21_
+
+_If you are upgrading: please see [`UPGRADING.md`](UPGRADING.md)._
+
+### Spark
+
+- **Breaking:** `Progressbar` has been moved to another package (`01e3a64`)
+```
+
+Use `> [!CAUTION]` and `> [!WARNING]` for important notices that may impact users like breaking changes, being from compilation or ui.
+- Example:
+```markdown
+> [!CAUTION] > The `Filled` style has been removed and may break your build if used.
+> 
+```
+
+#### Remove noise
+Exclude maintenance changes that are not interesting to consumers of the project (in its distributed form). To name a few:
+- Dotfile changes (`.gitignore`, `.github`, `.gitlab` and so forth)
+- Changes to development-only dependencies.
+- Minor code style changes.
+- Formatting changes in documentation.
+- Change on the CI that don’t affect the final product.
+
+However, changes such as the following must _not_ be excluded:
+- Refactorings, which may have unintentional side effects. Let the community review them.
+- Changes to supported runtime environments (which may be reflected only in dotfiles).
+- Code style changes that use new language features.
+- New documentation (if a feature was previously undocumented).
+
+---
+
+## Example Entry
+
+```markdown
+## [0.11.0]
+
+_2024-06-24_
+
+### Spark
+
+#### 🆕 Chips can now be selectable and closed
+> [!CAUTION]
+> The `Filled` style has been removed and may break your build if used. You need to see with your UI team to know which styles to use instead of this one.
+
+> [!WARNING]
+> The styles for chips have been deprecated; you now need to use either the `Chip` or the `ChipSelectable` components and provide the style in the argument.
+
+If you want to make your Chip closable then you will need to add a callback action in the new `onClose` parameter.
+
+---
```

Comment:
These separators are not documented, also, they are used to split content, so using it like this would not convey the proper meaning I think. We should probably get rid of them.

Reply:
You're right I'm removing them, I didn't noticed it before but h2 headers already add a divider in Github

Reply:
Done in 6730295